src/main/java/com/reservation/attendance/domain/AttendanceId.java

```diff
@@ -0,0 +1,6 @@
+package com.reservation.attendance.domain;
+
+import java.util.UUID;
+
+record AttendanceId(UUID id) {
```

Comment:
Null check mógłby być w konstruktorze:
```suggestion
record AttendanceId(UUID id) {
    public AttendanceId {
        Objects.requireNonNull(id);
    }
```